.github/workflows/formatting-linting.yml

```diff
@@ -17,4 +17,5 @@ jobs:
       - name: Run flake8
         run: flake8
       - name: Run black
-        run: black --check .
+        run: black --check --verbose --extend-exclude=".resources." .   
```

Comment:
In Confluence zu Linting and Formatting dazupacken, wie man files excluded. Und maybe as das --verbose bedeutet

Reply:
Zu Confluence hinzugefügt